arcade/raffle/build/imports/puzzle_arcade_ticket_v002.aleo

```diff
@@ -52,15 +52,70 @@ function spend:
 function join:
     input r0 as PuzzleArcadeTicket.record;
     input r1 as PuzzleArcadeTicket.record;
-    assert.eq self.caller self.signer;
-    is.eq r0.owner self.signer into r2;
+    gt r0.amount 0u64 into r2;
     assert.eq r2 true;
-    is.eq r1.owner self.signer into r3;
+    gt r1.amount 0u64 into r3;
+    assert.eq r3 true;
+    add r0.amount r1.amount into r4;
+    cast self.signer r4 into r5 as PuzzleArcadeTicket.record;
+    output r5 as PuzzleArcadeTicket.record;
+
+
+function join3:
```

Comment:
what are all these changes here? is this just as a result of the build files not being updated?

Reply:
prob just the fact that the imported ticket contract went from v1 to v2?